builtin/credential/github/backend_test.go

```diff
@@ -26,7 +26,7 @@ func TestBackend_Config(t *testing.T) {
 
 	login_data := map[string]interface{}{
 		// This token has to be replaced with a working token for the test to work.
-		"token": "4fb5f4f0738c7aea5ee06dd595f15236ea78918b",
+		"token": os.Getenv("GITHUB_TOKEN"),
```

Comment:
It's related in that I was trying to ensure the acceptance tests worked since the GitHub backend is one of the ones most affected by this change. The static token isn't valid, so the acceptance tests currently *always* fail. `GITHUB_TOKEN` is already required for the acceptance tests; this just uses that same token value to attempt to run this test.